ydb/core/tx/schemeshard/schemeshard__operation_create_external_table.cpp

```diff
@@ -393,8 +393,53 @@ class TCreateExternalTable: public TSubOperation {
 
 namespace NKikimr::NSchemeShard {
 
-ISubOperation::TPtr CreateNewExternalTable(TOperationId id, const TTxTransaction& tx) {
-    return MakeSubOperation<TCreateExternalTable>(id, tx);
+TVector<ISubOperation::TPtr> CreateNewExternalTable(TOperationId id, const TTxTransaction& tx, TOperationContext& context) {
```

Comment:
Yes - this method must return vector because of how it will be used in factory method later on. Please see `ISubOperation::TPtr TOperation::RestorePart(TTxState::ETxType txType, TTxState::ETxState txState)` at `schemeshard__operation.cpp` for more details